CONTRIBUTING.md

```diff
@@ -75,7 +76,13 @@ export PATH="<path_to_torchaudio>/third_party/install/bin:${PATH}"
 The following dependencies are also needed for testing:
 
 ```bash
-pip install typing pytest scipy numpy parameterized
+pip install typing pytest scipy numpy parameterized pyyaml
+```
+
+Optional packages to install if you want to run related tests:
+
+```bash
+pip install librosa requests soundfile kaldi_io transformers fariseq
```

Comment:
The latest version of `fairseq` from `pip install` does not work at the moment. So please do not put it in the command line.

Reply:
Sure, I've updated it in the latest commit.